.github/workflows/release.yml

```diff
@@ -46,5 +46,4 @@ jobs:
       working-directory: clients/cobol-lsp-vscode-extension
       run: export NODE_PATH=`npm root -g`; release-it --ci --config ../../release/release-it-release.json
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        THE_KEY: ${{ secrets.TELEMETRY_INSTRUMENTATION_KEY }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think that at this point the `release` job produces a package without a telemetry key.